lib/nodes/addon/components/driver-azure/template.hbs

```diff
@@ -235,6 +331,22 @@
           {{t "nodeDriver.azure.nsg.helpText"}}
         </p>
       </div>
+      <div class="col span-6">
+        <label class="acc-label">
+          {{t "nodeDriver.azure.acceleratedNetworking.label"}}
+        </label>
+        <div>
+          {{input
+            type="checkbox"
+            checked=config.acceleratedNetworking
```

Comment:
It would be cool if this worked like the rke2 accelerated networking checkbox: disabled when the selected vm size doesn't support accelerated networking and config.acceleratedNetworking is `false`. That can be achieved by adding `disabled=(and (not config.acceleratedNetworking) (not selectedVmSizeSupportsAN))`

Reply:
Added the disabled prop and a tooltip to make sure it's clear why. 'Accelerated networking is only supported for certain VM sizes.'
<img width="556" alt="Screenshot 2022-12-19 at 3 53 17 PM" src="https://user-images.githubusercontent.com/20599230/208542374-f1148138-f9d2-41df-b798-9ad23bc0a42c.png">